static/app/views/partials/sidebar.html

```diff
@@ -1,7 +1,8 @@
 <ul class="nav nav-sidebar">
+
     <li class="dropdown">
         <a class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true"
-        aria-expanded="false" id="org-dropdown">Organizations&nbsp;<span class="caret"></span></a>
+        aria-expanded="false" id="org-dropdown">Change Organization&nbsp;<span class="caret"></span></a>
```

Comment:
Something odd is happening to my side bar, at the small screen it doesn't overflow but it does at the medium.